mklive.sh.in

```diff
@@ -417,9 +419,9 @@ install_packages
 print_step "Enabling services: ${SERVICE_LIST} ..."
 enable_services ${DEFAULT_SERVICE_LIST} ${SERVICE_LIST}
 
-if [ -n "${INCLUDE_DIRECTORY}" ];then
-    print_step "Copying directory structure into the rootfs: ${INCLUDE_DIRECTORY} ..."
-    copy_include_directory
+if [ -n "${INCLUDE_DIRS}" ];then
+    print_step "Copying directory structures into the rootfs: ${INCLUDE_DIRS} ..."
```

Comment:
The list of include directories can get a bit busy in the output, maybe they could be printed individually using `info_msg()` in the for loop of `copy_include_directories()` instead?